app/views/notifications/index.html.erb

```diff
@@ -62,9 +62,16 @@
           </div>
         <% end %>
       <% end %>
-      <%= render "notifications_list" %>
-      <% if @notifications.any? %>
-        <div id="notifications-pagination" data-pagination-path="/notifications/<%= params[:org_id].present? ? "#{params[:filter]}/#{params[:org_id]}" : params[:filter].to_s %>?page=<%= @notifications.last.id %>"></div>
+
+      <%= render "notifications_list", params: params %>
+
+      <%# not using "any?"" here because "notifications_list" already asks for ".size", a little optimization :) %>
+      <% if @notifications.size > 7 %>
```

Comment:
I changed this line because this button doesn't need to show up if there are fewer than one pagination worth of notifications (better UX for brand-new users)